### PR TITLE
Add profile photo w/ drop-down menu

### PR DESCRIPTION
**Pull request by palourde**
- Upgrade to angular-bootstrap 0.13.0
- Bugfix with siteTheme directive
- Add profile picture with drop-menu menu:

screen shot 2015-05-03 at 14 56 04

cc @amdprophet

### CODE REVIEW

**Comment by calebhailey** [comment]
@palourde I like it! :shipit:

File: partials/views/events.html [hunk 1]
```diff
@@ -58,7 +58,7 @@
                   <span ng-click="stash($event, event)" ng-if="user.canPost()">
                     <i class="fa fa-fw {{ event.acknowledged | getAckClass }}"></i>
                   </span>
-                  <span class="main check-name" tooltip-placement="top" tooltip-trigger tooltip-html-unsafe="Subscriptions<br /><small>{{event.check.subscribers|arrayToString}}</small>">{{ event.check.name }}</span>
+                  <span class="main check-name" tooltip-placement="top" tooltip-trigger tooltip-html="'Subscriptions<br /><small>{{event.check.subscribers|arrayToString}}</small>'">{{ event.check.name }}</span>
```
Comment (amdprophet) [inline partials/views/events.html]: Does embedded HTML work within `tooltip-html`? I thought `-unsafe` was necessary for processing.

Comment (palourde) [inline partials/views/events.html]: `tooltip-html-unsafe` has been [deprecated](https://github.com/angular-ui/bootstrap/pull/3496) so we can now use `tooltip-html` to display embedded HTML and use `$sce.trustAsHtml()`as required.

**Comment by amdprophet** [comment]
:+1:

**Pull request merged**